features/step_definitions/admin_auction_form_steps.rb

```diff
@@ -35,6 +35,51 @@
   select("reverse", from: "auction_type")
 end
 
+When(/^I fill auction form with choosing "([^"]*)" purchase card$/) do |purchase_card|
+  @title = 'This is the form-edited title'
+  fill_in("auction_title", with: @title)
+
+  @description = 'and the admin related stuff'
+  fill_in("auction_description", with: @description)
+
+  @repo = 'https://github.com/18F/calc'
+  fill_in('auction_github_repo', with: @repo)
+
+  @summary = 'The Summary!'
+  fill_in('auction_summary', with: @summary)
+
+  @issue_url = 'https://github.com/18F/calc/issues/255'
+  fill_in('auction_issue_url', with: @issue_url)
+
+  @start_day = DcTimePresenter.convert(3.days.from_now)
```

Comment:
let's just leave `start_day` `end_day` and `delivery_deadline` as the defaults for this step -- we want to fill out all required fields but those have defaults so we can save ~10 lines of spec code by not filling those out. Sound OK to you?